### DISCUSSION TITLE
close Modals components

### DISCUSSION THREAD

**Discussion by leessmin** [skeletonlabs/skeleton#1992] [General]
I have encountered a problem,What should I do when I don't want to use the default event to handle modal shutdown for me?
For example, I don't want to trigger on: backdrop.

**Comment by endigo9740** [comment]
You can choose to close or clear the modal store queue at any time:
https://www.skeleton.dev/utilities/modals#close

We do not support modifying the behavior of the backdrop:
https://www.skeleton.dev/utilities/modals#accessibility

**Reply by adamdva** [reply]
The WAI-ARIA Roles does appear to allow marking a dialog modal is some cases. If skeleton modals do not support modifying the behavior of the backdrop, it can experience severe negative ramifications while working with dialog forms.

**Reply by endigo9740** [reply]
@adamdva I'm not sure why you chose to post this here. It's completely off topic to a month old thread.

If you find a gap in in our component ARIA roles, I'd encourage you to file a ticket in the Issues section. This is a discussion section, but Issues are actionable items we'll aim to resolve asap. Please consider posting there instead.

**Reply by adamdva** [reply]
@endigo9740 Sorry, since this thread was about 'close Modals components', I figured it was on topic to this recent thread, and wanted to add my disappointment to the topic that appeared closed by your comment. I may add an issue, but am currently working on an alternative dialog.
